software/apache-http-server.yml

```diff
@@ -1,6 +1,6 @@
 name: "Apache HTTP Server"
 website_url: "https://httpd.apache.org/"
-source_code_url: "https://svn.apache.org/viewvc/httpd/httpd/branches/2.4.x/"
+source_code_url: "https://github.com/apache/httpd"
```

Comment:
The correct URL is https://svn.apache.org/repos/asf/httpd/httpd/trunk/ (https://httpd.apache.org/dev/devnotes.html)

The github project is just a mirror

Reply:
I see i see then 👍🏻 I just wanted to future proof it a little in case the url breaks again (which hopefully does not in this case)